index.html

```diff
@@ -294,12 +294,12 @@
             A HTTP <code>PUT</code> request that includes an unsupported <code>Digest</code> type (as described
             in [[!RFC3230]]), SHOULD be rejected with a 400 Bad Request response.
           </p>
-          <p>
-            Implementations MUST support <code>Content-Type: message/external-body</code> extensions
-            for request bodies for HTTP <code>PUT</code> to <a>LDP-NR</a>s. This content-type requires a
-            complete <code>Content-Type</code> header that includes the location of the external body, e.g
-            <code>Content-Type: message/external-body; access-type=URL; URL=\"http://www.example.com/file\"</code>,
-            as defined in [[!RFC2017]].
+          <p class='informative'>
+            Implementations may support <code>Content-Type: message/external-body</code> extensions for
```

Comment:
For non-normative sections (such as this), I think the use of "may" is correct here.